packages/charting/src/components/Legends/Legends.base.tsx

```diff
@@ -238,7 +241,7 @@ export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
         onBlur={onMouseOut}
         data-is-focusable={true}
       >
-        <div className={classNames.rect} />
+        <div className={legend.shape === 'triangle' ? classNames.triangle : classNames.rect} />
```

Comment:
><div className={legend.shape === 'triangle' ? classNames.triangle : classNames.rect} /> [](start = 8, length = 87)

Can we think of a better way so that this is extensible?

Reply:
Sure. Encapsulate this part of the code as a function so we can easily extend it.

---
In reply to: [324826991](https://github.com/OfficeDev/office-ui-fabric-react/pull/10455#discussion_r324826991) [](ancestors = 324826991)